templates/default/web.xml.erb

```diff
@@ -1,18 +1,22 @@
 <web-app xmlns="http://java.sun.com/xml/ns/javaee" metadata-complete="true" version="2.5" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-	<display-name>/rundeck-production-2.6.4</display-name>
+	<display-name>/rundeck-production-2.6.10</display-name>
```

Comment:
Also I am thinking is there a need of custom web.xml anymore? In that case we can just drop that part of the code